src/material-experimental/mdc-chips/_chips-theme.scss

```diff
@@ -29,21 +29,36 @@
       @include mdc-chip-fill-color-accessible($unselected-background,
         $query: $mat-theme-styles-query);
 
+      // mdc-chip-fill-color-accessible includes mdc-chip-selected-ink-color which overrides the opacity
+      // so selected chips always show a ripple.
+      // Include the same mixins but use mdc-chip-selected-ink-color-without-ripple
       &.mat-primary {
         &.mdc-chip--selected, &.mat-mdc-chip-highlighted {
-          @include mdc-chip-fill-color-accessible($primary, $query: $mat-theme-styles-query);
+          @include mdc-chip-fill-color($primary, $query: $mat-theme-styles-query);
+          @include mdc-chip-ink-color(text-primary-on-dark, $query: $mat-theme-styles-query);
+          @include mdc-chip-selected-ink-color-without-ripple_(text-primary-on-dark, $query: $mat-theme-styles-query);
+          @include mdc-chip-leading-icon-color(text-primary-on-dark, $query: $mat-theme-styles-query);
+          @include mdc-chip-trailing-icon-color(text-primary-on-dark, $query: $mat-theme-styles-query);
```

Comment:
Optional: it would be nice if you could pull these 4 mixins out into another mixin so we don't have repeat as much. E.g.

```
@mixin _selected-color {
  @include mdc-chip-fill-color;
  @include mdc-chip-selected-ink-color-without-ripple_
  ....
}
```